CRM/Core/Permission/Standalone.php

```diff
@@ -48,7 +48,58 @@ class CRM_Core_Permission_Standalone extends CRM_Core_Permission_Base {
    *   true if yes, else false
    */
   public function check($str, $userId = NULL) {
-    return \Civi\Standalone\Security::singleton()->checkPermission($this, $str, $userId);
+    // These core-defined synthetic permissions (which cannot be applied by our Role UI):
+    // cms:administer users
+    // cms:view user account
+    // need mapping to our concrete permissions (which can be applied to Roles) with the same names:
+    $str = $this->translatePermission($str, 'Standalone', [
+      'view user account' => 'view user account',
+      'administer users' => 'administer users',
+    ]);
```

Comment:
@artfulrobot can you help me understand why sometimes the name of the permission is prefixed with "cms:" in this PR and other times it isn't?

I'm especially confused by this array which purports to map from one style to the other but... doesn't.

Reply:
Thanks @artfulrobot that makes sense! I've added https://github.com/civicrm/civicrm-core/pull/28835 to try to make that function easier for me (and other devs who stumble across it in future) to understand.